snooty/test_parser.py

```diff
@@ -3527,3 +3527,30 @@ def test_invalid_icon() -> None:
     )
     page.finish(diagnostics)
     assert len(diagnostics) == 0
+
+
+def test_openapi_changelog() -> None:
```

Comment:
We don't actually need tests for rstspec-only changes since there are no new code paths being introduced :D

Reply:
:) I was wondering! But I felt like better safe than sorry. I'll delete!